Purchases.xcodeproj/project.pbxproj

```diff
@@ -1032,12 +1032,12 @@
 			path = Networking;
 			sourceTree = "<group>";
 		};
-		35F82BBB26A9BFA60051DF03 /* Misc */ = {
+		35F82BBB26A9BFA60051DF03 /* FoundationExtensions */ = {
```

Comment:
just move it from the `Misc` folder to the existing `FoundationExtensions` folder